gtsfm/utils/ellipsoid.py

```diff
@@ -31,11 +31,15 @@ def get_ortho_axis_alignment_transform(gtsfm_data: GtsfmData) -> Pose3:
 
     # Filter outlier points, Center point cloud, and obtain alignment rotation.
     points_filtered = remove_outlier_points(point_cloud)
-    points_centered, mean = center_point_cloud(points_filtered)
+    points_centered = center_point_cloud(points_filtered)
     wuprightRw = get_alignment_rotation_matrix_from_svd(points_centered)
 
+    # Calculate translation vector based off rotated point cloud.
```

Comment:
I realized the translation vector that I'd been using in the final `walignedTw` pose was incorrect. Previously, I was using the `-1 * mean` of the **original point cloud** and using that as the translation. However, I realized that first rotation is applied, then translation. Thus, the correct translation to use is `-1 * mean` of the **rotated point cloud** (so that after rotation is applied, I can center it).

Reply:
nice, can you show what it looks like both ways?

Reply:
With the incorrect translation, the point cloud renders as:
![image](https://user-images.githubusercontent.com/40643713/139556437-616b3b42-0521-40a3-b8e3-82deecb5b02b.png)

With the correct translation vector, the point cloud renders as:
![image](https://user-images.githubusercontent.com/40643713/139556455-c0c8ac96-594e-4a1f-94f5-99562b2e32c8.png)